lib/page/house_wikidata.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../../lib/popolo_helper.rb'
+
+module Page
+  class HouseWikidata
+    def initialize(slug, house)
+      @house = house
+      @slug = slug
+    end
```

Comment:
Our style here is also to put a private `attr_reader` on the instance variables, and have all access go through the methods, rather than direct variable access.